src/Sulu/Bundle/ContactBundle/Resources/public/dist/components/contact-form/main.js

```diff
@@ -1 +1 @@
all(this)),this.sandbox.emit(k.call(this)),void P.call(this)):!1},P=function(){this.sandbox.dom.off(d.bankAccountFormId)},Q=function(){this.sandbox.dom.off(d.addressFormId),this.sandbox.stop(d.addressTypeSelector)},R=function(a){var b,c;return this.sandbox.form.validate(d.addressFormId)?(b=this.sandbox.form.getData(d.addressFormId,!0),a?(this.sandbox.form.editInCollection(this.form,a,b),c=this.$find('[data-mapper-id="'+a+'"]'),U.call(this,c)):this.sandbox.form.addToCollection(this.form,"addresses",b).then(function(a){U.call(this,a)}.bind(this)),this.sandbox.emit(h.call(this)),this.sandbox.emit(i.call(this)),void Q.call(this)):!1},S=function(){var a,b,c={};this.dropdownDataArray=[],this.$addOverlay=this.sandbox.dom.createElement(e.add),this.sandbox.util.foreach(this.options.fields,function(b,d){if(!this.options.fieldTypes||!this.options.fieldTypes[b])throw"contact-form@sulu: fieldTypes not defined for type "+b;a={id:d,name:this.sandbox.translate("public."+b),type:b,collection:b+"s",items:this.options.translatedFieldTypes[b]},c[b]=a}.bind(this)),c.fax.collection="faxes",this.dropdownDataArray=Object.keys(c).map(function(a){return c[a]}),b=this.sandbox.dom.createElement("<div>"),this.sandbox.dom.append(this.$el,b),this.sandbox.start([{name:"overlay@husky",options:{el:b,title:this.sandbox.translate("public.add-fields"),openOnStart:!0,removeOnClose:!0,instanceName:"add-fields",okInactive:!0,data:this.$addOverlay,okCallback:C.bind(this)}},{name:"dependent-select@husky",options:{el:this.$addOverlay,singleSelect:!0,data:this.dropdownDataArray,defaultLabels:this.sandbox.translate("public.please-choose"),deselectField:this.sandbox.translate("select.no-choice"),instanceName:"add-fields",container:["#"+d.fieldId,"#"+d.fieldTypeId],isNative:!0}}])},T=function(){var a,b,c=this.sandbox.dom.find("label.hidden","#contact-edit-form");for(a=-1,b=c.length;++a<b;)U.call(this,this.sandbox.dom.parent(c[a]))},U=function(a){var b=this.sandbox.dom.trim(this.sandbox.dom.text(this.sandbox.dom.find("label.hidden",a)));b=b.replace(/\s{2,}/g," "),this.sandbox.dom.attr(this.sandbox.dom.find("label.visible",a),"title",b),this.sandbox.dom.html(this.sandbox.dom.find("label.visible",a),this.sandbox.util.cropMiddle(b,20))};return{initialize:function(){this.initializedFlag=!1,this.$editOverlayContent=null,this.form=null,this.$addOverlay=null,this.dropdownDataArray=[],this.editFieldsData=[],this.data=null,this.options=this.sandbox.util.extend(!0,{},c,this.options),H.call(this),this.render(),n.call(this),o.call(this),this.sandbox.emit(g.call(this)),this.initializedFlag=!0},render:function(){var a=this.sandbox.dom.createElement('<div id="contact-form-options-container" />'),b=this.$find(d.dropdownContainerId);this.sandbox.dom.append(b,a),this.sandbox.start([{name:"dropdown@husky",options:{trigger:b,triggerOutside:!0,el:a,alignment:"right",shadow:!0,toggleClassOn:b,data:[{id:1,name:"public.edit-fields",callback:L.bind(this)},{id:2,name:"public.add-fields",callback:S.bind(this)}]}}])}}});
```

Comment:
I did not change it. I think someone forgot to build it and therefore it's in here.